app/frontend/components/pages/sendAda/multiAssetsPage.tsx

```diff
@@ -71,36 +111,19 @@ const MultiAssetsPage = () => {
               )
               const details = (
                 <div className={`${styles.details} ${isExpanded ? styles.expanded : ''}`}>
-                  <div className="multi-asset-page-policy">
-                    <CopyOnClick
-                      value={asset?.policyId}
-                      elementClass="copy"
-                      tooltipMessage="Policy id copied to clipboard"
-                    >
-                      <div className="multi-asset-page-copy-on-click">
-                        <span className={styles.hashLabel}>Policy ID:</span>
-                        {formattedPolicy}
-                        <div className="desktop">
-                          <span className="copy-text margin-left" />
-                        </div>
-                      </div>
-                    </CopyOnClick>
-                  </div>
-                  <div className="multi-asset-page-policy">
-                    <CopyOnClick
-                      value={asset?.fingerprint}
-                      elementClass="copy"
-                      tooltipMessage="Fingerprint copied to clipboard"
-                    >
-                      <div className="multi-asset-page-copy-on-click">
-                        <span className={styles.hashLabel}>Fingerprint:</span>
-                        {formattedFingerprint}
-                        <div className="desktop">
-                          <span className="copy-text margin-left" />
-                        </div>
-                      </div>
-                    </CopyOnClick>
-                  </div>
+                  <Item title="Name" displayElement={formattedOffChainName!} />
+                  <Item title="Ticker" displayElement={formattedTicker!} />
+                  <Item
+                    title="Policy ID"
+                    displayElement={formattedPolicy!}
+                    copyValue={asset.policyId}
+                  />
+                  <Item title="Asset name" displayElement={formattedOnChainName!} />
+                  <Item
+                    title="Fingerprint"
+                    displayElement={formattedFingerprint!}
+                    copyValue={asset.fingerprint!}
```

Comment:
I'd find it convenient to add a link here to the block explorer so the user can check there details about the token and gain confidence they are indeed sending the right one

Reply:
on the other hand, the user can do so from the list of assets they hold, and it's more natural to do it from there, so after a second thought, there isn't that much added value to it indeed